extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -33,6 +33,12 @@ const POSITION_FIXED_TAG_WHITELIST = {
   'AMP-STICKY-AD': true,
 };
 
+/** @const These tags are valid ad containers */
+const AMP_AD_CONTAINERS = {
```

Comment:
Good point. We can reuse the list to keep things simple. If later there's outlier, we can split the list at that time.